avaje-config/src/main/java/io/avaje/config/Configuration.java

```diff
@@ -287,6 +290,14 @@ public interface Configuration {
    */
   void setProperty(String key, String value);
 
+  /**
+   * Add configuration values via a map. Note that {@link #eventBuilder(String)} should be
```

Comment:
I think the javadoc note about "eventBuilder() should be used" is confusing because this uses eventBuilder under the hood.

Reply:
ok